ci-operator/step-registry/gcp/provision/vpc/gcp-provision-vpc-commands.sh

```diff
@@ -50,6 +50,14 @@ cat > "${SHARED_DIR}/vpc-destroy.sh" << EOF
 gcloud deployment-manager deployments delete -q "${CLUSTER_NAME}-vpc"
 EOF
 
+if [[ "${DISCONNECTED_NETWORK}" = "yes" ]]; then
+  echo "Updating the VPC into a disconnected network (removing NAT and enabling Private Google Access)..."
+  gcloud compute routers nats delete -q "${CLUSTER_NAME}-nat-master" --router "${CLUSTER_NAME}-router" --region "${REGION}"
+  gcloud compute routers nats delete -q "${CLUSTER_NAME}-nat-worker" --router "${CLUSTER_NAME}-router" --region "${REGION}"
+  gcloud compute networks subnets update "${CLUSTER_NAME}-master-subnet" --region "${REGION}" --enable-private-ip-google-access
+  gcloud compute networks subnets update "${CLUSTER_NAME}-worker-subnet" --region "${REGION}" --enable-private-ip-google-access
```

Comment:
`--enable-private-ip-google-access` seems like it might be something we want for these subnets in all cases, not just these blackholed/restricted-network/disconnected setups.  Are there downsides to turning this on for more-connected VPCs?

Reply:
By default, `--enable-private-ip-google-access` is not enabled by default when creating a vpc, also our official document did not mention that, so we want to keep consistent behavior like what customers do.

Reply:
maybe we should add it to our official documentation?

Reply:
Per our testing, it indeed isn't a MUST, just some special case (blackhole network) needs it, should we always asking customers to enable it? And in the present official document, we did not place a detailed explanation about `restricted network`, also did not mention how to pre-create a VPC per different platforms, I guess that is because that is a bit out of scope of OCP install, also different customers have different use cases.